MultiServer.py

```diff
@@ -1419,6 +1431,11 @@ def get_hints(self, input_text: str, for_location: bool = False) -> bool:
                             hints.extend(collect_hints(self.ctx, self.client.team, self.client.slot, item_name))
                 elif not for_location and hint_name in self.ctx.item_names_for_game(game):  # item name
                     hints = collect_hints(self.ctx, self.client.team, self.client.slot, hint_name)
+                elif hint_name in self.ctx.location_name_groups[game]:  # location group name
+                    hints = []
+                    for loc_name in self.ctx.location_name_groups[game][hint_name]:
+                        if loc_name in self.ctx.location_names_for_game(game):
+                            hints.extend(collect_hint_location_name(self.ctx, self.client.team, self.client.slot, loc_name))
```

Comment:
This looks like it will hint all of the locations in a group even if the location is not in the seed.
Is that right?
Would it be better to hint only locations that exist in this seed?
(I think that's in `ctx.locations`)
...either that or make the location groups instance variables rather than class variables, so they can be defined for each seed?

Reply:
I see now `collect_hint_location_id` (called by `collect_hint_location_name`) checks `ctx.locations` to make sure the location exists in this seed.